neo/Network/P2P/Connection.cs

```diff
@@ -66,18 +66,49 @@ private void WsReceive()
                 failure: ex => new Tcp.ErrorClosed(ex.Message));
         }
 
-        public void Disconnect(bool abort = false)
+        protected void Disconnect(byte[] lastMessage = null)
```

Comment:
`lastMessage` should be moved to `RemoteNode`.

Reply:
fixed